test/3x/torch/algorithms/fp8_quant/unit_tests/test_functions/test_config_json.py

```diff
@@ -26,10 +26,8 @@ def test_config_json():
             QuantMode.MEASURE: "measure",
             QuantMode.QUANTIZE: "quant",
         }[mode]
-        config_path = os.path.join(
-            os.environ.get("NEURAL_COMPRESSOR_FORK_ROOT"),
-            f"neural_compressor/torch/algorithms/fp8_quant/custom_config/llama_{name}.json",
-        )
+        config_path = f"llama_{name}.json"
```

Comment:
relying on relative path might not work since it will attempt to find the path from quant_config,py file which may be the installed package path.

Reply:
Thanks @MrGeva, we also noticed this issue. We applied below change to make sure that `json` files are saved in the installed packages. The UT has passed the test now.
https://github.com/intel/neural-compressor/pull/2025/commits/b072f18515a23b249a2df63a3b55244b9c486a16